singer-io/tap-google-ads
# Discover reports

# Description of change
Adding report discovery

# Manual QA steps
 - Ran tap and checked UI to see that reports showed up and tables and fields could be selected along with field exclusions
 
# Risks
 - None
 
# Rollback steps
 - revert this branch
